### PR TITLE
bug 1409321, clean up django commands, r?mathjazz

### PR DESCRIPTION
**Pull request by Pike**
Also remove some left-over __init__.py files that didn't have
commands anymore.
And remove importbuilds, which we used to use to bootstrap
elmo build status from buildbot. We won't do that anymore, so
that command can just go away.

Ride-along adding some ldap groups to pick up by the django
management command that inspects ldap permissions.

### CODE REVIEW

**Comment by Pike** [comment]
Matjaz, would you be up to review this? Just plain django-related changes, no functional stuff, really.

File: lib/auth/backends.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
     # Django name:  LDAP name(s),
     'Localizers': ('scm_l10n', 'locale_peer'),
     'build': ('buildteam', 'shipit'),
+    'l10n-drivers': ('scm_l10n_drivers',),
+    'l10n-infra': ('scm_l10n_infra',)
```
Comment (Pike) [inline lib/auth/backends.py]: This breaks the tests, and would also break auth on prod, I'll back that out.

I want to get rid of one of the two anyway.

File: apps/bugsy/management/commands/process_bugogram.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ class Command(BaseCommand):
     props = re.compile('^; (.*?) : (.*?)$', re.M | re.S)
     params = re.compile('%\((.*?)\)s')
 
-    def handle(self, *args, **options):
+    def handle(self, **options):
```
Comment (mathjazz) [inline apps/bugsy/management/commands/process_bugogram.py]: What's the rationale for removing `*args`?

Comment (Pike) [reply apps/bugsy/management/commands/process_bugogram.py]: I read somewhere that it's deprecated, but I forgot which version/doc that was in.

In https://github.com/django/django/blob/stable/1.11.x/django/core/management/__init__.py#L127, it shows that *args is only filled if you actually create a named argparse argument `args`. All others end up as keyword arguments now.

**Pull request merged**